src/nos/core/notifications.py

```diff
@@ -180,6 +180,13 @@ def formatted(self, **kwargs) -> "Notification":
 """,
 )
 
+IVA_SEND_CODE_TRANSMISSION = Notification(
+    "IVA Verification Code",
+    """
+Your verification code is: {code}
```

Comment:
We could also add recipient or sender to the text, but I agree it's better to keep it short. If we add a proper Sender ID, the recipient should see that it is from GHGA.

Reply:
Yes, the sender ID is added. I kept it short so it will (hopefully) be fully displayed in the push notification, and you won’t need to open the messaging app.